src/integrationTest/java/uk/gov/hmcts/reform/bsp/clients/BlobRouterServiceClientTest.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.bsp.clients;
+
+import jdk.jfr.Description;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
+import uk.gov.hmcts.reform.bsp.config.feign.BlobRouterServiceClient;
+import uk.gov.hmcts.reform.bsp.integrations.SlackClient;
+import uk.gov.hmcts.reform.bsp.models.SearchResult;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(properties = {
+    "app.enabled=true",
+    "app.trigger-type=BULK_SCAN_CHECKS"
+})
+@Description("Check the integration between Blob Router works as expected. "
+    + "Note that the abort and create endpoints are not checked here, as the service once deployed "
+    + "Will be removing them all twice daily.")
+class BlobRouterServiceClientTest {
+
+    @Autowired
+    private BlobRouterServiceClient client;
+
+    @MockitoBean
+    private SlackClient slackClient;
+
+    @DisplayName("Should delete all stale blobs and return successfully")
+    @Test
+    void deleteAllStaleBlobsTest() {
+        SearchResult<String> resp = client.deleteAllStaleBlobs(168);
```

Comment:
Not sure I agree with having a test that calls the delete all blobs endpoint, unless its a stub/mocked but doesn't look to be that way

Reply:
The delete blobs one is fine I think, as its only for AAT and since it returns an empty list if all is fine it wont fail for that reason and still checks the connection/response

with reference to the configured client

Although for the other delete/updates and so forth, I don't think it possible at present